topydo/ui/CompleterBase.py

```diff
@@ -0,0 +1,92 @@
+# Topydo - A todo.txt client written in Python.
+# Copyright (C) 2017 Bram Schoenmakers <bram@topydo.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import datetime
+
+from topydo.Commands import _SUBCOMMAND_MAP
```

Comment:
Yep, I know that, but it was also imported with underscore previously (cfr. `TopydoCompleter.py` in `master` branch), so I thought you left it intentionally :)